onko_dicom.py

```diff
@@ -13,13 +13,17 @@
 )
 import plot_widget
 
-logging.basicConfig(
-    filename='hellopyside.log',
-    filemode='w',
-    level=logging.INFO,
-    format='%(asctime)s.%(msecs)03d %(levelname)s:%(message)s',
-    force=True
-)
+LOG_FILES_DIR = 'logs'
+if not os.path.isdir(LOG_FILES_DIR):
+    os.makedirs(LOG_FILES_DIR)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+formatter = logging.Formatter('%(asctime)s.%(msecs)03d %(levelname)s:%(name)s:%(message)s')
+file_handler = logging.FileHandler('logs/onko_dicom.log', mode='w')
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
```

Comment:
is there a better way to do this via the logging.basicConfig()?

Reply:
No, logging basic config touches on the root logger, this creates a logger that isn't root allows for diff files and/or settings

There is however prob a better way to do this such as having a logger.py file with that as a method and return a logger, yet to test this idea though

Reply:
ah okay seems good for now then